armi/materials/material.py

```diff
@@ -729,7 +724,9 @@ class Fluid(Material):
     """A material that fills its container. Could also be a gas."""
 
     def getThermalExpansionDensityReduction(self, prevTempInC, newTempInC):
-        """Return the factor required to update thermal expansion going from temperatureInC to temperatureInCNew."""
+        """Return the factor required to update thermal expansion going from temperatureInC to
+        temperatureInCNew.
```

Comment:
this docstring doesn't match the parameters

Reply:
Fixed